src/components/MemberList/MemberList.vue

```diff
@@ -79,44 +76,43 @@
 			@close="resetPicker"
 			@search="onSearch"
 			@submit="onPickerPick" />
-	</AppContentList>
+	</section>
 </template>
 
-<script>
+<script lang="ts">
 import {
-	NcAppContentList as AppContentList,
-	NcButton as Button,
-	NcEmptyContent as EmptyContent,
-	NcLoadingIcon as IconLoading,
+	NcButton,
+	NcEmptyContent,
+	NcLoadingIcon,
 	isMobile,
 } from '@nextcloud/vue'
 
-import MembersListItem from './MembersList/MembersListItem.vue'
-import EntityPicker from './EntityPicker/EntityPicker.vue'
+import MemberListGroup from './MemberListGroup.vue'
+import EntityPicker from '../EntityPicker/EntityPicker.vue'
 import IconContact from 'vue-material-design-icons/AccountMultiple.vue'
 import IconAdd from 'vue-material-design-icons/Plus.vue'
-import IconInfo from 'vue-material-design-icons/InformationOutline.vue'
-import RouterMixin from '../mixins/RouterMixin.js'
+import RouterMixin from '../../mixins/RouterMixin.js'
 
-import { getRecommendations, getSuggestions } from '../services/collaborationAutocompletion.js'
 import { showError, showWarning } from '@nextcloud/dialogs'
 import { subscribe } from '@nextcloud/event-bus'
-import { SHARES_TYPES_MEMBER_MAP, CIRCLES_MEMBER_GROUPING } from '../models/constants.ts'
+import { t } from '@nextcloud/l10n'
```

Comment:
```suggestion
import { translate as t } from '@nextcloud/l10n'
```

Reply:
or update `@nextcloud/l10n`?

Reply:
If necessary yes, otherwise I'd keep the package where it is for this stable branch.